mixins/StoreMixin.js

```diff
@@ -76,9 +78,15 @@ var StoreMixin = {
   store_addListeners: function () {
     Object.keys(this.store_listeners).forEach(function (storeID) {
       var listenerDetail = this.store_listeners[storeID];
+      var events = listenerDetail.events;
 
+      // Check that we actually have events to fire events on
+      if (typeof events !== 'object' || !Object.keys(events).length) {
```

Comment:
What if we wrap this `if` in a `if (process.env === 'development') {`, because when we compile this for release it'll replace the `process.env` with `'production'` so it's a super fast check.